### ISSUE TITLE
Missing test_policy_ppo.py

### ISSUE THREAD

**Issue by BitastaB** [openai/spinningup#354]
It should be under utils. Any idea how or where can I get this file?

**Comment by pythlang** [comment]
I was also looking for this. I just renamed the call test_policy and it was working decently

**Comment by BitastaB** [comment]
Are you talking about the MACL project? For me, it fails in testing.

**Comment by pythlang** [comment]
Yes. I believe so. I was looking for it; I had to tweak some other file import paths also for it to run properly but I got it to at least run

**Comment by BitastaB** [comment]
For me it breaks at Line 87: get_action = lambda x : sess.run(action_op, feed_dict={model['x']: x[None,:]})[0]. I tried tweaking it, but couldn't make it run :(

**Comment by pythlang** [comment]
What is the error?

**Comment by BitastaB** [comment]
That model does not have key x. Did you do something else? I can see that load_policy_and_env function of the same file was also changed in this project because he expects some lstm value along with  others in return. He also passes this lstm to run_policy which does not match with original params.

**Comment by BitastaB** [comment]
@pythlang sorry for bothering you, but do you have any suggestions for me?

**Comment by alan303138** [comment]
> @pythlang sorry for bothering you, but do you have any suggestions for me?

Has the problem been solved?I got same question.
